src/components/Content/ContentList.css

```diff
@@ -4,34 +4,25 @@
   @mixin remove-a;
 }
 
-.element {
-  height: 50px;
-}
-
-.element-text {
-  height: 100%;
-  min-width: 1px; /* exact value doesn't matter, but this lets long words wrap */
-  flex-grow: 1;
-  margin-left: 15px;
-  padding: 15px 0;
-  border-bottom: 2px solid #fbda16;
-  word-wrap: break-word;
-}
-
 .row {
   display: flex;
   align-items: center;
   margin-bottom: 10px;
 }
 
-.element-image {
+.category-thumbnail {
   width: 50px;
+  height: 50px;
   flex-shrink: 0;
   padding: 5px;
 }
 
-.image {
-  width: 50px;
-  max-width: 100%;
-  max-height: 100%;
+.category-caption {
+  height: 100%;
+  min-width: 1px; /* needed to enabled line breaks for to long words, exact value doesn't matter, @Max: DO NOT CHANGE */
+  flex-grow: 1;
+  margin-left: 15px;
+  padding: 15px 0;
+  border-bottom: 2px solid #fbda16;
```

Comment:
Please change `border-bottom: 2px solid #fbda16;` to `border-bottom: 2px solid var(--theme-color);` and add `@import url('../Common/Colors.css');` at the top of the file

Reply:
done, merged develop into WEBAPP-4